android/expoview/src/main/res/layout/simple_spinner_dropdown_item.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.reactnativecommunity.picker.CheckedTextViewImpl xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@tsapeta i think I had to add these files to get `bare-expo` to build on CI - its confusing because these files are not in previous versions but previous versions also reference them - any ideas why that is?

Reply:
That's fine, these two xmls just override the default ones provided by Android's SDK (see https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/layout/simple_spinner_dropdown_item.xml). Looks like previously the library just used the default implementation, but now it has its own implementation in `CheckedTextViewImpl` and `TextViewImpl` classes

Reply:
ahhh that makes sense! thank you!